infrastructure/testdriver/set_timezone.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver set_time_zone method</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+
+<script>
+function defaultTimeZone() {
+  return (new Intl.DateTimeFormat()).resolvedOptions().timeZone;
+}
+async_test(t => {
+  let timeZone = "Asia/Taipei";
+  let timeZone2 = "Asia/Hong_Kong";
```

Comment:
These are both GMT+8 (i.e. same local time). Can we use two timezones that actually differ to make the difference more clear? Say, `'Asia/Taipei'` and `'Europe/Berlin'` (GMT+1), or `'Pacific/Fakaofo'` (GMT+13) as you use in the other tests.

```suggestion
  const timeZone = "Asia/Taipei";
  const timeZone2 = "Pacific/Fakaofo";
```

WDYT?